playground/infrastructure/checker.py

```diff
@@ -99,7 +99,7 @@ def main():
     if check_sdk_examples(args.paths, Sdk.Value(args.sdk), root_dir):
         return
 
```

Comment:
exit code 1 can mask away checker.py exceptions, need to choose a somewhat more rare one